### DISCUSSION TITLE
Add bot commands to suggested commands in my server

### DISCUSSION THREAD

**Discussion by loimax** [Rapptz/discord.py#9149] [Q&A]
Hi, I'd like my bot commands to be displayed when i type a forward slash in my discord server, here : 
![image](https://user-images.githubusercontent.com/90707356/209193379-763bd71f-aa75-48b2-86ee-abf5e70b9a45.png)
I'll join my code in here, I'm kind of new to all of this, sorry in advance for the state of the code ^^
Best regards.
![image](https://user-images.githubusercontent.com/90707356/209193866-66d4137e-3b24-4571-a9f5-6ea409381266.png)
![image](https://user-images.githubusercontent.com/90707356/209193930-0a9115c1-6dbf-4596-99ee-e7bb51aea59b.png)
![image](https://user-images.githubusercontent.com/90707356/209193970-8111ab8c-2d03-4fa1-ab47-23b12c2a346a.png)
![image](https://user-images.githubusercontent.com/90707356/209193997-fc995cef-afe1-45c9-a35e-c768008f8023.png)
![image](https://user-images.githubusercontent.com/90707356/209194015-c871b423-c594-4c5d-8ad3-895e9e160dd4.png)

**Comment by imvaskel** [comment]
The easiest way to do this is to convert your `@commands.command` into `@commands.hybrid_command`. See the docs [here](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.hybrid_command). Basically, this class turns your command into both a text and application (slash) command. It should be noted however, that you will need to do more than just dropping in the decorator (notably you must sync your command tree to have them appear).

**Reply by loimax** [reply]
Hi, thanks for the quick answer ! I implemented what you told me but get an error that I don't quite understand : 
![image](https://user-images.githubusercontent.com/90707356/209203125-36ae0f28-d14e-45e9-a3a4-88294d6229ab.png)
I implemented it that way : 
![image](https://user-images.githubusercontent.com/90707356/209203458-a6abfffa-d56d-4553-911e-3ea06443d31c.png)

**Reply by loimax** [reply]
I'll try to sync it now as you said, i'll update you later.

**Reply by loimax** [reply]
![image](https://user-images.githubusercontent.com/90707356/209207430-0099d881-a41b-4080-aa14-cab51c32eee8.png)
I did this to sync my commands and it still doesn't seem to work (found this on an online tutorial).

**Reply by imvaskel** [reply]
Your top errors is due to the fact that you are doing `*args` which I believe is unsupported. As for your second issue, it's a bit hard to debug here so I'd recommend joining the [discord](https://discord.gg/dpy) for better help.

**Reply by loimax** [reply]
Okay, thanks a lot for your answers ! I'll join the discord then.
